source/upgrade/_sysvm_templates_pre45.rst

```diff
@@ -18,8 +18,9 @@
 Update System-VM templates
 --------------------------
 
-.. note::
-   Upgrading pre-4.3 to 4.4.1 require 2 systemvm templates downloaded: the 4.3 and 4.4.
+.. warning::
+   Upgrading from 4.4 or older to 4.6.0 require 2 systemvm templates to be
```

Comment:
This file ``_sysvm_templates_pre45.rst`` is not used in the RN, I left it there as reference.